.github/workflows/deployment.yaml

```diff
@@ -19,7 +19,7 @@ permissions:
   packages: write
 jobs:
   terraform:
-    name: Run Terraform (${{github.event.inputs.environment}})
```

Comment:
This can be put back if we want, but I removed it since I was just doing true CI/CD to dev2 while developing and this variable didn't have an value at the time.